articles/enroll-personal-byod-ios-ipad-hosts-with-managed-apple-account.md

```diff
@@ -0,0 +1,66 @@
+# Enroll personal (BYOD) iPhones and iPads with Managed Apple Account
+
+_Available in Fleet Premium._
+
+To enroll personal iPhones and iPads using [Account-driven User Enrollment](https://support.apple.com/en-gb/guide/deployment/dep23db2037d/web), and provide a seamless experience for your end users, follow the steps below.
+
+This approach enables both a Managed Apple Account and a personal Apple Account to be signed in on the same device, ensuring complete separation of work profile and personal data. Users retain privacy over their personal information, while IT manages work-related OS settings and applications.
```

Comment:
```suggestion
With Account-driven User Enrollment end users can separate work and personal data using their [Managed Apple Accounts](https://support.apple.com/en-gb/guide/apple-business-manager/axm78b477c81/web). End users retain privacy over their personal information, while IT admins manage work-related OS settings and applications.
```

Copy tweaks. Emphasize the why (separate data) which you explained well.

Thoughts?